src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -422,6 +425,37 @@ def setPassword(self, newPassword):
         else:
             raise Exception('Unknown error '+ hex(receivedPacketPayload[0]))
 
+    def handShake(self):
+        """
+        handshake the fingerprint sensor.
```

Comment:
Please document the information you provided in the PR also here in method documentation:
* For which sensor it is
* What is the purpose of the call
* What means the result